includes/definitions/truenas.yaml

```diff
@@ -25,4 +25,5 @@ discovery:
             - .1.3.6.1.4.1.50536.3
         sysDescr_regex:
             - '/freenas/i'
-            - '/truenas/i'
+            - '/^(?!.*TrueNAS-SCALE).*$/i'
```

Comment:
This seems wrong as it doesn't detect core right?

Reply:
Because the entire expression searches for strings that begin and end, and that at no point in between contain the exact sequence "TrueNAS-SCALE" this should detect core. 
Here are some examples where TrueNAS(CORE) and FreeNAS matches and SCALE doesn't: https://regex101.com/r/09xtcP/3

Reply:
I don't think we want that, it works now but could have unintended effects in the future. Better to match what is known now then include everything known and unknown.

Reply:
Hint, this is probably why you are matching dell-compellent

Reply:
Great! I'll update it: It should contain TrueNAS but it shouldn't contain TrueNAS-SCALE https://regex101.com/r/09xtcP/4